src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -35,6 +35,11 @@ var EventInterface = {
   defaultPrevented: null,
   isTrusted: null,
 };
+/**
+ * These are additional properties not in the EventInterface
+ * which are nulled when destructoring a syntheticEvent instance
+ */
+var otherNullableProps = ['dispatchConfig', '_targetInst', 'nativeEvent'];
```

Comment:
`dispatchConfig` and `_targetInst` are both implementation details and are considered private fields.
I think we don't have to warn on accessing those.

This leaves us with `nativeEvent` which can be hardcoded as a special case below.